docs/cow-protocol/tutorials/swap-in-safe-app/README.mdx

```diff
@@ -13,6 +13,22 @@ This is interesting for Safe Dapps because:
 
 The easiest way to get started is to follow the steps below.
 
+## DEMO
+Before you start, you can get a feel of the Safe App example. Feel free to skip this step!
+
+The example shows:
+- ✅ How to make a new Safe App.
+- ✅ How to make Swaps, Limit orders and TWAPs.
+- ✅ How to control programatically the swap widget.
+- ✅ How to detect trade parameters (token selection, amounts, etc.).
+- ✅ How to define a dynamic fee.
+- ✅ How to customize the colors, tokens, sounds, etc.
```

Comment:
```suggestion
- How to make a new Safe App
- How to make Swaps, Limit orders and TWAPs
- How to control programatically the swap widget
- How to detect trade parameters (token selection, amounts, etc.)
- How to define a dynamic fee
- How to customize the colors, tokens, sounds, etc
```

As these aren't full sentences, bullet points don't have full stops at the end per style guide. Also, we are generally avoiding the use of emojis excluding top-level navigation items (i.e. CoW Protocol, MEV Blocker etc).